starter-code/css/main.css

```
@@ -6,3 +6,136 @@ Green: #00B172
 White: #FFF
 
 */
+
+#main-logo{
+    position: absolute; top: 15px; left: 40px;
+    width: 20%;
+}
+
+#menu {
+ background-color: white;
+ width: 100%;
+ height: 60px; 
```

Aquí tienes un padding que te genera un scroll innecesario, lo eliminas poniendo un padding
```suggestion
 height: 60px; 
 padding: 0;
```